tests/cli_integration.rs

```diff
@@ -48,29 +51,29 @@ async fn all_commands() {
 		);
 	}
 
-	// Export to stdout
+	info!("* Export to stdout");
 	{
 		let args = format!("export --conn http://{addr} {creds} --ns N --db D -");
 		let output = common::run(&args).output().expect("failed to run stdout export: {args}");
 		assert!(output.contains("DEFINE TABLE thing SCHEMALESS PERMISSIONS NONE;"));
 		assert!(output.contains("UPDATE thing:one CONTENT { id: thing:one };"));
 	}
 
-	// Export to file
+	info!("* Export to file");
 	let exported = {
 		let exported = common::tmp_file("exported.surql");
 		let args = format!("export --conn http://{addr} {creds} --ns N --db D {exported}");
 		common::run(&args).output().expect("failed to run file export: {args}");
 		exported
 	};
 
-	// Import the exported file
+	info!("* Import the exported file");
```

Comment:
Why are we changing these to info level logs? That means we get an INFO level formatted log entry when the args are constructed, right? So users always see that

Reply:
given we run several test cases inside a single test function, I found it useful to receive a bit more feedback about what's going on inside it. The original use case was for debugging a test run that hang, so with these log messages I was able to see which test case was the culprit